### PR TITLE
Fix ocp4 cluster dns

### PR DESCRIPTION
**Pull request by wkulhanek**
(and a minor bugfix for authentication workload with LDAP)

### CODE REVIEW

File: ansible/configs/ocp4-cluster/default_vars_azure.yml [hunk 1]
```diff
@@ -50,3 +50,7 @@ master_instance_type: Standard_D8s_v3
 # Machine Type for worker nodes
 worker_instance_type: Standard_D4s_v3
 
+# admin keys. Azure specific. Other platforms get these from the bastion roles
+opentlc_admin_pub_keys:
+  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC3Avw03Dmh1R2QWQ4CV7JgEsXnHQjNhfppD5aZmh0q/64p6lW+2oNKTT7fVQcrsdmlJwrMd5apkUGrOcq0hHXQMEVZEKUmEjko2BqD5A9/zNX7apObW88bFFfgxc91lOT+e+wfCFsrr3b2SJ3+KL6nTBJV7Lf46i6z86vhiDPjqL7U9kTS+bK9ldU20vpn8h+ZAIaiafVWfjihUjhNpcUY46klixV1YcAkBGCbE+YR6RAAc6vWy0zB3YJnTUl9OFt213ofi1qjuWKVMmOxORxPKB4/JQ+hfAsCMysoVFnFYs10dWxaySK63OgY9uLNyaIwkEaVVIfcViRVm0DZfoNH gucore
```
Comment (stencell) [inline ansible/configs/ocp4-cluster/default_vars_azure.yml]: We can probably remove GCs key from every default we have...

File: ansible/configs/ocp4-cluster/default_vars_osp.yml [hunk 2]
```diff
@@ -33,20 +32,24 @@ ansible_user: cloud-user
...
+ocp4_base_domain: "{{ osp_cluster_dns_zone }}"
 
 # The dynamic DNS server you will add entries to.
 # NOTE: This is only applicable when {{ use_dynamic_dns}} is true
-osp_cluster_dns_server: ddns01.opentlc.com
-
-# Whether to wait for an ack from the DNS servers before continuing
-wait_for_dns: true
+# This needs to come from secrets
+# osp_cluster_dns_server: ddns01.opentlc.com
```
Comment (stencell) [inline ansible/configs/ocp4-cluster/default_vars_osp.yml]: I think this was moved to secrets and can properly be something like `FROMSECRET` here?

File: ansible/configs/ocp4-cluster/files/requirements_k8s.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 adal==1.2.2
-ansible==2.9.4
+ansible==2.9.6
```
Comment (stencell) [inline ansible/configs/ocp4-cluster/files/requirements_k8s.txt]: Just making sure, you have moved all the previously named `k8s` and `os` modules from their 2.8 names to 2.9 names?

Comment (wkulhanek) [reply ansible/configs/ocp4-cluster/files/requirements_k8s.txt]: This is the latest pip freeze list.

File: ansible/configs/ocp4-cluster/default_vars_osp.yml (hunk 2 above)
Comment (wkulhanek) [reply ansible/configs/ocp4-cluster/default_vars_osp.yml]: Good point.

File: ansible/configs/ocp4-cluster/default_vars_azure.yml (hunk 1 above)
Comment (wkulhanek) [reply ansible/configs/ocp4-cluster/default_vars_azure.yml]: Yeah.

**Pull request merged**